cosmos/plugin/__init__.py

```diff
@@ -1,273 +1,11 @@
-import os.path as op
```

Comment:
all of the existing content of this file has been copied over as-is to the new `plugin_impl.py` file in the same directory path

Reply:
Do you know why Git didn't consider this a rename, but instead treated it as a new file?

Perhaps we could rename this just to `plugin/plugin.py` - its a bit ungly, but I feel impl is not adding much. Happy with alternatives as well :)